app/src/strategies/tokenStrategy.js

```diff
@@ -0,0 +1,22 @@
+// @flow
```

Comment:
briefly describe the purpose of this file

Reply:
Done